pilot/pkg/security/authn/v1alpha1/policy_applier.go

```diff
@@ -144,6 +145,27 @@ func convertToEnvoyJwtConfig(policyJwts []*authn_v1alpha1.Jwt) *envoy_jwt.JwtAut
 		providers[name] = provider
 	}
 
+	providerNames := make([]string, 0, len(providers))
```

Comment:
I think you can actually just create the `requirementOrList` in the beginning of the function and append the `JwtRequirement` just after line 145 to avoid the extra for-loop, the sort() call and the providerNames.

Reply:
done